include/net/ethernet.h

```diff
@@ -375,6 +383,13 @@ struct ethernet_context {
 	s8_t vlan_enabled;
 #endif
 
+#if defined(CONFIG_NET_DSA)
+	/** Switch physical port number */
+	u8_t dsa_port_idx;
```

Comment:
revert the order of these 2 attributes. And move the whole before line 370, for better alignement

Reply:
Why shall I move this code before line 370? This will not provide alphabetic order in any way?